<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoreBasicSettings</class>
 <widget class="QWidget" name="MoreBasicSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Basic Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="SettingAndInfo_Layout" stretch="1,99">
     <item>
      <layout class="QVBoxLayout" name="SettingBox_Layout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="Port">
         <item>
          <widget class="QLabel" name="Port_Label">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="text">
            <string>Port:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="font">
            <font>
             <family>Segoe UI</family>
            </font>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="Port_Refresh">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Icons/Icons.qrc">
             <normaloff>:/Spin_1.png</normaloff>:/Spin_1.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="Port_Combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox{
background:rgba(255,255,255,255);
border-radius:4px;
border:1px solid rgba(217,217,217,255);
}
QComboBox::drop-down {
    border: 0px solid rgb(255,255,255);
    subcontrol-origin:padding;
    padding-right: 10px;
}

QComboBox::down-arrow {
	image: url(:/downCommbox.png);
    width:16px;
    height:16px;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Baudrate">
         <item>
          <widget class="QLabel" name="BaudRate_Name">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="text">
            <string>BondRate:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="font">
            <font>
             <family>Segoe UI</family>
            </font>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="BaudRate_LineEdit">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
	border-width:0;
	border-style:outset;
	
	selection-background-color: rgb(85, 85, 255)
	
}

QLineEdit:hover
{
	border: 1px solid blue
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Databits">
         <item>
          <widget class="QLabel" name="DataBits_Label">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="text">
            <string>Data bits:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="font">
            <font>
             <family>Segoe UI</family>
            </font>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="DataBits_Combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox{
background:rgba(255,255,255,255);
border-radius:4px;
border:1px solid rgba(217,217,217,255);
}
QComboBox::drop-down {
    border: 0px solid rgb(255,255,255);
    subcontrol-origin:padding;
    padding-right: 10px;
}

QComboBox::down-arrow {
	image: url(:/downCommbox.png);
    width:16px;
    height:16px;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Parity">
         <item>
          <widget class="QLabel" name="Parity_label">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="text">
            <string>Paritys:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="font">
            <font>
             <family>Segoe UI</family>
            </font>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="Parity_Combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox{
background:rgba(255,255,255,255);
border-radius:4px;
border:1px solid rgba(217,217,217,255);
}
QComboBox::drop-down {
    border: 0px solid rgb(255,255,255);
    subcontrol-origin:padding;
    padding-right: 10px;
}

QComboBox::down-arrow {
	image: url(:/downCommbox.png);
    width:16px;
    height:16px;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="FlowControl">
         <item>
          <widget class="QLabel" name="FlowControl_Label">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="text">
            <string>Flow Control</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="font">
            <font>
             <family>Segoe UI</family>
            </font>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="FlowControl_Combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox{
background:rgba(255,255,255,255);
border-radius:4px;
border:1px solid rgba(217,217,217,255);
}
QComboBox::drop-down {
    border: 0px solid rgb(255,255,255);
    subcontrol-origin:padding;
    padding-right: 10px;
}

QComboBox::down-arrow {
	image: url(:/downCommbox.png);
    width:16px;
    height:16px;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Stopbits">
         <item>
          <widget class="QLabel" name="StopBits_Label">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="text">
            <string>Stop bits:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="font">
            <font>
             <family>Segoe UI</family>
            </font>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="StopBits_Combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox{
background:rgba(255,255,255,255);
border-radius:4px;
border:1px solid rgba(217,217,217,255);
}
QComboBox::drop-down {
    border: 0px solid rgb(255,255,255);
    subcontrol-origin:padding;
    padding-right: 10px;
}

QComboBox::down-arrow {
	image: url(:/downCommbox.png);
    width:16px;
    height:16px;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="PortDetail_TreeWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <column>
        <property name="text">
         <string>Port</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Details</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Manufacturer</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Cancel_PushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{

}
QPushButton:hover{

}
</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PortOpenClose_PushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{

}
QPushButton:hover{

}
</string>
       </property>
       <property name="text">
        <string>Open Port</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Icons/Icons.qrc"/>
  <include location="../Icons/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
